### PR TITLE
CF-128 Additional address type 'Postal and Visiting'

### PR DESCRIPTION
**Pull request by cgiHarland**
Provide a new address type so that a single address can be shown for Postal and Visiting.

Change does not require admins to remove other types.

Tests included.

### CODE REVIEW

File: data/test_data/courts.json [hunk 1]
```diff
@@ -447,7 +447,206 @@
         "contacts": [],
         "postcodes": ["SA79RB"],
         "info": null,
-        "hide_aols": false,
+        "hide_aols": false
```
Comment (richerry) [inline data/test_data/courts.json]: Missing comma makes json sad.

**Comment by cgiHarland** [comment]
2 attempts but the json is fixed.

**Comment by richerry** [comment]
test_data/courts.json conflicting again :( maybe because I just pulled PR#177

**Comment by cgiHarland** [comment]
Fixed json.

**Pull request merged**